lib/vanagon/engine/always_be_scheduling.rb

```diff
@@ -37,7 +37,21 @@ def name
 
       # return the platform name as the "host" name
       def build_host_name
-        @platform.name
+        platform_name_to_vmpooler_style(@platform.name)
+      end
+
+      def platform_name_to_vmpooler_style(platform)
+        translation = {
+          "aix-5.3-ppc" => "aix-53-ppc",
```

Comment:
Or add another field to the platform definition (essentially what `vmpooler_template` is doing for pooler-based platforms)?

```ruby
platform "..." do |plat|
  ...
  plat.abs_resource_name "aix-53-ppc"
end
```